cypress/support/config/services.js

```diff
@@ -873,6 +873,30 @@ module.exports = {
       mediaAssetPage: { path: undefined, smoke: false },
     },
   },
+  scotland: {
+    font: undefined,
+    isWorldService: false,
+    variant: 'default',
+    pageTypes: {
+      articles: {
+        path:
+          Cypress.env('APP_ENV') === 'live' || Cypress.env('APP_ENV') === 'test'
+            ? undefined
+            : '/scotland/articles/c1111111111o',
```

Comment:
This should be updated to point to an article that exists locally in the application.
```suggestion
            : '/scotland/articles/czwj5l0n210o',
```

Reply:
Oops. My mistake - I changed the ID of the fixture while working.